react/src/components/carousel/TextboxItem.module.css

```diff
@@ -0,0 +1,143 @@
+/* Universal Layout */
+:root {
+    /* Descriptive variables */
+    --color-darkBlue: #1883b4;
```

Comment:
Please declare colours in one css file, and use them everywhere. If it is special colour just for special case, then it should not be in `:root` and variable name should describe that **special** case.

Reply:
Regarding the name, that's what I aimed for with the `Functional variables` underneath. Should I do it differently?